accounts/store_kvdb.go

```diff
@@ -244,6 +244,33 @@ func (s *BoltStore) CreditAccount(_ context.Context, id AccountID,
 	return s.updateAccount(id, update)
 }
 
+// DebitAccount decreases the balance of the account with the given ID
+// by the given amount.
+//
+// NOTE: This is part of the Store interface.
+func (s *BoltStore) DebitAccount(_ context.Context, id AccountID,
+	amount lnwire.MilliSatoshi) error {
+
+	update := func(account *OffChainBalanceAccount) error {
+		if amount > math.MaxInt64 {
+			return fmt.Errorf("amount %v exceeds the maximum of %v",
+				amount, int64(math.MaxInt64))
+		}
```

Comment:
think we can move this check out of the db transaction since it does not need access to the account

Reply:
Thanks :pray:! Addressed with the latest push.